src/Microsoft.Data.Analysis/DataFrame.cs

```diff
@@ -278,15 +253,15 @@ public DataFrame OrderByDescending(string columnName)
         /// <typeparam name="U"></typeparam>
         /// <param name="lower">Minimum value. All values below this threshold will be set to it</param>
         /// <param name="upper">Maximum value. All values above this threshold will be set to it</param>
-        public DataFrame Clip<U>(U lower, U upper, bool inPlace = false)
+        public DataFrame Clamp<U>(U lower, U upper, bool inPlace = false)
```

Comment:
Maybe `lower` => `min` and `upper` => `max`?

Reply:
Yup. Changed the parameter names in the `Filter` too